xml/deploy_core.xml

```diff
@@ -800,6 +883,135 @@ placement:
     defining NFS exports through the NFS section in the &dashboard;:
    </para>
 <screen>&prompt.cephuser;ceph mgr module enable nfs</screen>
+   <sect3 xml:id="ceph-nfs-ha">
+    <title>Deploying &ha; for the &ganesha;</title>
+    <para>
+     You can create a high availability (HA) endpoints for the &ganesha;
+     service by deploying &ingress;. &ingress; includes a combination of
+     &haproxy; and &keepalived; to provide load balancing on a floating virtual
+     IP address. A virtual IP address is automatically configured on only one
+     of &ingress; hosts at a time.
+    </para>
+    <para>
+     Each &keepalived; daemon checks every few seconds whether the &haproxy;
+     daemon on the same host is responding. &keepalived; also checks whether
+     the master &keepalived; daemon is running without problems. If the master
+     &keepalived; or the active &haproxy; daemon is not responding, one of the
+     remaining &keepalived; daemons running in backup mode is elected as the
+     master daemon and the virtual IP address will be assigned to that node.
+    </para>
+    <figure>
+     <title>&ha; setup for &ganesha;</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata fileref="ses-nfs-ha.png" width="100%"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="ses-nfs-ha.png" width="100%"/>
+      </imageobject>
+      <textobject role="description"><phrase>&ha; setup for &ganesha;</phrase>
+      </textobject>
+     </mediaobject>
+    </figure>
+    <procedure>
+     <step>
+      <para>
+       Ensure that the following sysctl variables are set on each &ingress;
+       host:
+      </para>
+<screen>
+net.ipv4.ip_nonlocal_bind=1
+net.ipv4.ip_forward=1
+</screen>
+      <tip>
+       <para>
+        You can persistently set the sysctl variables by creating a file
+        <filename>/etc/sysctl.d/90-ingress.conf</filename> and including the
+        above variables. To apply the changes, run <command>sysctl
+        --system</command> or reboot the host.
+       </para>
+      </tip>
+     </step>
```

Comment:
this is a bug that has been fixed upstream and will appear in the next v16.2.11 release. perhaps a TID would be more appropriate?